### PR TITLE
chore(build): remove unused dependencies

### LINKED ISSUE #4: Remove gulp-duration
The `gulp-duration` package is no longer used.

### PR DESCRIPTION
**Pull request by mewdriller**
* Remove `gulp-duration` and other unused dependencies from `package.json`
 * Add `tearDown` to the component spec example
 * Add another `.eslintrc` in the `test` directory to add `jasmine` env

Closed #4, #5, #6

### CODE REVIEW

**Comment by mewdriller** [comment]
@tborres: Let me know what you think of this updated setup.

File: test/services/sample/sample.service.spec.js [hunk 1]
```diff
@@ -18,7 +34,6 @@ describe('SampleService', () => {
   });
 
   describe('.delayedLog', () => {
-
```
Comment (tborres) [inline test/services/sample/sample.service.spec.js]: Have you decided to no longer place spaces at the beginning and end of describe blocks?

**Comment by tborres** [comment]
Looks good.

File: test/services/sample/sample.service.spec.js (hunk 1 above)
Comment (mewdriller) [inline test/services/sample/sample.service.spec.js]: That's part of the `eslint` configuration from Airbnb apparently, so yeah.

**Pull request merged**